Packs/1_1_3SuperchargingXSOARWithAnsible/Integrations/VMwareV2/VMwareV2.yml

```
@@ -0,0 +1,4551 @@
+category: IT Services
+commonfields:
+  id: VMwareV2
+  version: -1
+configuration:
+- additionalinfo: The hostname or IP address of the vSphere vCenter or ESXi server.
+  display: Hostname
+  name: hostname
+  required: true
+  type: 0
+- additionalinfo: The port of the vSphere vCenter or ESXi server.
+  defaultvalue: "443"
+  display: Port
+  name: port
+  required: true
+  type: 0
+- additionalinfo: The username to access the vSphere vCenter or ESXi server.
+  display: Username
+  name: username
+  required: true
+  type: 0
+- additionalinfo: The password to access the vSphere vCenter or ESXi server.
+  display: Password
+  name: password
+  required: true
+  type: 4
+- additionalinfo: Allows connection when SSL certificates are not valid. Set to `false`
+    when certificates are not trusted.
+  defaultvalue: "True"
+  display: Validate Certs
+  name: validate_certs
+  required: true
+  type: 8
+description: Manage VMware vSphere Server, Guests, and ESXi Hosts
+display: VMware V2
```

```suggestion
display: VMware v2
```
Should be lower case according to our convention.